### PR TITLE
86: Fixes default state of Add Agency UX 

### PR DESCRIPTION
**Pull request by User1m**
### Description

Fixes default state of Add Agency UX FROM ....

Screen Shot 2022-06-17 at 3 29 41 PM

TO...

### Screenshots / Demo Video

Screen Shot 2022-06-17 at 3 25 00 PM
Screen Shot 2022-06-17 at 3 26 11 PM
Screen Shot 2022-06-17 at 3 26 20 PM
Screen Shot 2022-06-17 at 3 26 26 PM
Screen Shot 2022-06-17 at 3 26 38 PM
Screen Shot 2022-06-17 at 3 29 34 PM
Screen Shot 2022-06-17 at 3 29 41 PM

### Testing
* Click Agency > Click "Add"

### Checklist
- [x] Provided ticket and description
- [x] Your PR title contains the ISSUE ticket number e.g "86: ..."
- [x] Provided screenshots/demo
- [x] Provided testing information
- [x] Provided adequate test coverage for all new code
- [x] Added PR reviewers
- [x] Ensure at least 1 review before merging

### CODE REVIEW

**Comment by mhuang-usdr** [comment]
I would set the default Warning threshold to 14 days and default Danger Threshold to 7 days. Otherwise LGTM.

File: packages/client/src/components/Modals/AddAgency.vue [hunk 1]
```diff
@@ -107,8 +105,8 @@ export default {
       formData: {
         name: null,
         abbreviation: null,
-        warningThreshold: null,
-        dangerThreshold: null,
+        warningThreshold: 2,
```
Comment (mhuang-usdr) [inline packages/client/src/components/Modals/AddAgency.vue:108]: Set to 14

File: packages/client/src/components/Modals/AddAgency.vue [hunk 2]
```diff
@@ -107,8 +105,8 @@ export default {
...
+        dangerThreshold: 1,
```
Comment (mhuang-usdr) [inline packages/client/src/components/Modals/AddAgency.vue:109]: set to 7

**Review by mhuang-usdr — CHANGES_REQUESTED** [review]

File: packages/client/src/components/Modals/AddAgency.vue (hunk 1 above)
Comment (mhuang-usdr) [reply packages/client/src/components/Modals/AddAgency.vue:108]: Yes, please set the default warning threshold to 14 days, and default danger threshold to 7 days.

**Comment by User1m** [comment]
@mhuang-usdr changes made. please approve when you can. TY!

**Review by mhuang-usdr — CHANGES_REQUESTED** [review]
Apologies, I think I wasn't clear earlier. Here are the product requirements: 

- For Warning Threshold:
  - minimum should be 2
  - default should be 14
- For danger Threshold:
  - minimum should be 1
  - default should be 7

**Comment by User1m** [comment]
@mhuang-usdr  updates made. Thanks for clarifying

**Review by mhuang-usdr — APPROVED** [review]

**Pull request merged**